cmd/release/cmd/tag.go

```diff
@@ -279,6 +280,49 @@ var dashboardTagSubCmd = &cobra.Command{
 	},
 }
 
+var cliTagSubCmd = &cobra.Command{
+	Use:   "cli [ga,rc,alpha,test] [version]",
+	Short: "Tag dashboard releases",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return errors.New("expected at least two arguments: [ga,rc,alpha,etc] [version]")
+		}
+
+		version := args[1]
+		if _, found := rootConfig.CLI.Versions[version]; !found {
+			return errors.New("verify your config file, version not found: " + version)
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		releaseType := args[0]
+
+		rc, err := releaseTypePreRelease(releaseType)
+		if err != nil {
+			return err
+		}
+
+		tag := args[1]
+		ctx := context.Background()
+		ghClient := repository.NewGithub(ctx, rootConfig.Auth.GithubToken)
+
+		cliRelease, found := rootConfig.CLI.Versions[tag]
+		if !found {
+			return errors.New("verify your config file, version not found: " + tag)
```

Comment:
This error appears to be used here more than once which means it should be made a const or value of some sort. Double check if it's used elsewhere and we should make a var or const that holds it.

E.g.

```go
// errVersionNotFound is used when a given version can't be found, generally, in config. The
// expectation of the caller is to append the version that's not found to the end of this value.
var errVersionNotFound = errors.New("verify your config file, version not found: ")
```

Reply:
Did a bit different implementation, but the results should be the same:

```golang
type VersionNotFoundError struct {
	Version string
}

func (e *VersionNotFoundError) Error() string {
	return fmt.Sprintf("verify your config file, version not found: %s", e.Version)
}

func NewVersionNotFoundError(version string) error {
	return &VersionNotFoundError{
		Version: version,
	}
}
```

So this can be called like this:
```golang
k3sRelease, found := rootConfig.K3s.Versions[version]
if !found {
	return NewVersionNotFoundError(version)
}
```

Implemented it in all files within `cmd/release/cmd/*`